exercises/saddle-points/src/main/java/MatrixCoordinate.java

```diff
@@ -1,31 +0,0 @@
-class MatrixCoordinate {
```

Comment:
Please don't remove this file. Like the occasional enum definitions and exception classes in other exercises, this has been provided as part of the solution so that the exercise isn't overly complicated to implement :) You also don't need a `.keep` file if you keep this in.